_tour/package-objects.md

```diff
@@ -62,11 +104,53 @@ object PrintPlanted {
   }
 }
 ```
+{% endtab %}
+{% tab 'Scala 3' for=pkg-obj-vs-top-lvl_4 %}
+```
+// in file PrintPlanted.scala
+import gardening.fruits.*
+
+@main def PrintPlanted: Unit =
```

Comment:
This is a bit unusual. Main methods are usually written in lower case:

```suggestion
@main def printPlanted(): Unit =
```

On line 90, maybe you can say “the following program imports…” (without giving the name of the program)?

Reply:
that's a good suggestion! thanks